docs/weaviate/concepts/replication-architecture/cluster-architecture.md

```diff
@@ -124,6 +124,40 @@ If the cluster size is 3 and the replication factor is also 3, then all nodes ca
 
 If the cluster size is 10 and the replication factor is 3, the 3 nodes which contain that data (collection) can serve queries, coordinated by the coordinator node. The client waits until x (the consistency level) nodes have responded.
 
+## Replica movement
+
+:::info Added in `v1.32`
+:::
+
+Weaviate allows users to manually move or copy individual shard replicas from a source node to a destination node in a Weaviate cluster. This capability addresses operational scenarios such as cluster rebalancing after scaling, node decommissioning, optimizing data locality for improved performance, or increasing data availability.
+
+Replica movement operates as a state machine with stages that ensure data integrity throughout the process. The feature works for both single-tenant collections and multi-tenant collections. 
+
+Unlike the static replication factor configured at collection creation, replica movement allows the replication factor to be adjusted for specific shards as replicas are moved or copied across the cluster. When a copy operation is performed, the newly created replica increases the replication factor for that specific shard. While a collection may have a default replication factor, individual shards within that collection can temporarily have a higher replication factor. However, shards can't have a replication factor lower then the one set on the collection level. 
+
+### Movement states
+
+The replica movement process follows a coordinated workflow that maintains data consistency and availability. Each operation progresses through distinct states that reflect the current stage of the movement:
```

Comment:
It's not clear to me what a "process" vs "operation" is here.

Wdyt about:

> Each replica movement operation progresses through a workflow designed to maintain data consistency and availability. The workflow comprises of the following states:

Reply:
True, operation is more consistent than process